util/app-config/src/configs/network.rs

```diff
@@ -105,7 +116,14 @@ impl Config {
             .create(true)
             .write(true)
             .open(path)
-            .and_then(|mut file| file.write_all(&random_key_pair))
+            .and_then(|mut file| {
+                #[cfg(unix)]
+                {
+                    use std::os::unix::fs::PermissionsExt;
+                    file.set_permissions(fs::Permissions::from_mode(0o600))?;
+                }
+                file.write_all(&random_key_pair)
+            })
```

Comment:
Maybe we can use `std::fs::set_permissions` here, this function works on Windows. rust said platform-specific behavior may change in the future, but that's future.

Reply:
Do we need to make separate settings for non-UNIX systems? I think the `read_only` restriction is not appropriate. What I want here is that only owner can access